### PR TITLE
[73] Excessive number of scan jobs scheduled

### PR DESCRIPTION
**Pull request by keithchong**

### CODE REVIEW

File: dev/com.ibm.ws.st.liberty.buildplugin.integration/src/com/ibm/ws/st/liberty/buildplugin/integration/manager/internal/AbstractLibertyManager.java [hunk 1]
```diff
@@ -120,6 +120,8 @@ boolean isValidConfig(LibertyBuildPluginConfiguration config) {
 
     protected abstract boolean handleGenerationPrompt(String name);
 
+    protected abstract boolean isApplicableProjectToScan(IProject project);
```
Comment (eharris369) [inline dev/com.ibm.ws.st.liberty.buildplugin.integration/src/com/ibm/ws/st/liberty/buildplugin/integration/manager/internal/AbstractLibertyManager.java]: Can this be more generic like isSupportedProject()?

File: dev/com.ibm.etools.maven.liberty.integration/src/com/ibm/etools/maven/liberty/integration/manager/internal/LibertyManager.java [hunk 2]
```diff
@@ -44,4 +47,15 @@ protected boolean handleGenerationPrompt(String projectName) {
         return UIHelper.handleGenerationPrompt(LibertyMavenConstants.PROMPT_PREFERENCE, NLS.bind(Messages.generationPromptMsg, projectName));
     }
 
+    /** {@inheritDoc} */
+    @Override
+    protected boolean isApplicableProjectToScan(IProject project) {
+        try {
+            return project.hasNature(LibertyMavenConstants.MAVEN_PROJECT_NATURE);
+        } catch (CoreException e) {
+            Trace.trace(Trace.INFO, "Error getting project nature for" + project.getName(), e);
```
Comment (eharris369) [inline dev/com.ibm.etools.maven.liberty.integration/src/com/ibm/etools/maven/liberty/integration/manager/internal/LibertyManager.java]: Trace levels other than ERROR should be surrounded by if (Trace.ENABLED).  This looks more like a warning than an info as well.

**Comment by keithchong** [comment]
@eharris369 , comments have been addressed.  Please review/approve.

**Comment by keithchong** [comment]
fixes #73

**Pull request merged**